src/Workspaces/Core/Portable/Remote/WellKnownServiceHubServices.cs

```diff
@@ -16,13 +16,6 @@ internal static class WellKnownServiceHubServices
         public const string CodeAnalysisService_FindReferenceMethodsAsync = "FindReferenceMethodsAsync";
         public const string CodeAnalysisService_GetFullyQualifiedName = "GetFullyQualifiedName";
 
-        // NavigateTo
-        public const string CodeAnalysisService_SearchDocumentAsync = "SearchDocumentAsync";
```

Comment:
thses are not needed anymore.  I just use 'nameof' to ensure that the names stay in sync.